mirror_csi_images/scripts/publish.sh

```diff
@@ -1,14 +1,28 @@
 #!/bin/bash
 
 INFILE="${PWD}/infile"
-touch $INFILE
+touch "${INFILE}"
 
-IFS=, read -ra CSI_IMAGES_ARR <<< "${CSI_IMAGES}"
-for CSI_IMAGE in "${CSI_IMAGES_ARR[@]}"; do
-  IFS=: read -ra IMAGE_TAG_PAIR <<< "$CSI_IMAGE"
-  echo "k8s.gcr.io/sig-storage/${IMAGE_TAG_PAIR[0]}" "longhornio/${IMAGE_TAG_PAIR[0]}" "${IMAGE_TAG_PAIR[1]}" >> $INFILE
-done
+if [[ -n "${LONGHORN_IMAGES_FILE_URL}" ]]; then
+
+  LONGHORN_IMAGES_FILE=${PWD}/longhorn-images.txt
+  wget "${LONGHORN_IMAGES_FILE_URL}" -O "${LONGHORN_IMAGES_FILE}"
```

Comment:
note: This URL should point to a raw image txt to ensure this works correctly. https://raw.githubusercontent.com/longhorn/longhorn/master/deploy/longhorn-images.txt